src/go/rdctl/pkg/snapshot/manager.go

```diff
@@ -210,12 +212,18 @@ func (manager *Manager) Restore(name string) (err error) {
 	}
 	defer func() {
 		// Don't restart the backend if the restore failed
-		unlockErr := manager.Unlock(manager.Paths, err == nil)
+		unlockErr := manager.Unlock(manager.Paths, !errors.Is(err, ErrDataReset))
```

Comment:
if `err` is nil, `errors.Is(nil, ErrDataReset)` would be truthy, so `!errors.Is(…)` would be false?

Reply:
I don't think that's right. `errors.Is(nil, ErrDataReset)` evaluates to `false`, does it not? I wrote a test program to try it out:
```golang
package main

import (
	"errors"
	"fmt"
)

var testErr = errors.New("this is a test error")

func main() {
	fmt.Println(errors.Is(nil, testErr))
}
```

Reply:
Oh, you're right. Sorry, I have no idea what I was thinking…